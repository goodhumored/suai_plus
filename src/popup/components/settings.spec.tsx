import renderer from "react-test-renderer";
import Settings from "./settings";
import React from "react";

it("renders well", () => {
  const component = renderer.create(<Settings />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot("settings");
});
