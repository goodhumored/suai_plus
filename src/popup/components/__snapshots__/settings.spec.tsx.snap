// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders well: settings 1`] = `
<div>
  <h4>
    Настройки:
  </h4>
  <ul
    id="settings"
  >
    <li />
  </ul>
</div>
`;
